uncompress_test.go

```diff
@@ -55,3 +57,40 @@ func TestUncompressEmpty(t *testing.T) {
 	_, err = payload.Next()
 	assert.ErrorIs(t, err, io.EOF)
 }
+
+func TestUncompressSizesAndChecksums(t *testing.T) {
+	files := []string{
+		"testdata/crypto-policies-20210617-1.gitc776d3e.el8.noarch.rpm",
+		"testdata/glibc-all-langpacks-2.28-164.el8.aarch64.rpm",
```

Comment:
I suppose it might be undesirable to include this particular packages, as I see other tests use made-up packages, but presently it's not clear to me what is special about these, hopefuly in the process of fixing the bug it'll become clearer.